pkg/cmd/server/api/helpers.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+import (
+	"errors"
+	"io/ioutil"
+)
+
+func (i *FileOrData) GetData() ([]byte, error) {
```

Comment:
I vaguely remember us not wanting to define methods on any API objects, but I can't remember why. `GetData(i *FileOrData)` instead?